infrastructure/dpladm/vpa-templates/cli-vpa.template.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: cli-vpa
+  namespace: ${SITE_NAME}-${SITE_ENV}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: cli
+  updatePolicy:
+    updateMode: "Initial"
```

Comment:
I'd like to initially just deploy it with `Off` and then inspect the recommendations manually before having them applied.

Reply:
I did that in all the test projects on both their environments, and it looked robust and with good measure to me. 
I also tested on a couple of live sites. Do you still want them of for now?

Reply:
Yes, lets have the observations documented first. :) I'd hate for this to blow up in our face and not having had four eyes on the recommendations first.

Reply:
:+1: